allure-results/000428ab-c284-43be-9a38-f15f581179de-attachment.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Allure reports are bound conceptually to each test execution. They do not belong structurally to the repo (the repository should ideally contain only files that are necessary for running the project). Consider deleting them, then ignoring the whole `allure-results` directory

Reply:
Perfect I'll do it on the next push.
Thanks!

Reply:
Its done, now what do I do?

Reply:
Now I just resolve the comment after I see the new commit